src/topojson/topojson.ml

```diff
@@ -416,17 +423,39 @@ module Make (J : Intf.Json) = struct
                     (decode_or_err (J.to_array (decode_or_err J.to_float)))))
               arcs
           in
+
+          (*let* transform =
+            (*J.to_string transform in
+            let float=
+               (J.to_list(J.to_array(decode_or_err J.to_float)))
+              transform*)
+            J.to_obj in let float= (J.to_list(J.to_array(decode_or_err (J.to_float J.to_float))))
+              transform*)
+          let* transform = J.to_obj transform in
+
+          let float = (J.to_array (decode_or_err J.to_float)) transform in
```

Comment:
`transform` is an object here not a float array, you will need to pull out the two float arrays under the two keys. `float` is also not a great name for this variable.

Reply:
@patricoferris can this work to pull out the two float arrays under the two keys??
let* transform = 
            match {} with
            |  {scale; translate} -> string_of_float scale ^" '" ^ string_of_float translate;
            transform

Reply:
No, there's an example just above of accessing these values.

```ocaml
 match
  (J.find json [ "objects" ],
   J.find json [ "arcs" ],
   J.find json [ "transform" ] )
       with
       | Some objects, Some arcs, Some transform ->
```